ui/models/job.js

```diff
@@ -103,8 +104,8 @@ export default class JobModel {
       for (const [key, value] of Object.entries(uniquePerPushJobs)) {
         const decisionTaskId = taskIdMap[key].id;
 
-        TaskclusterModel.load(decisionTaskId, null, currentRepo).then(
-          async results => {
+        TaskclusterModel.load(decisionTaskId, null, currentRepo, testMode)
```

Comment:
I couldn't think of a better way to get the jobs_tests to pass without using the `testMode` arg (since the first time an action is called `taskcluster.getCredentials` will return null so this was a way to return mock credentials with `taskcluster.getMockCredentials`. If someone has a better idea, let me know!

Reply:
This seems reasonable.  I can sympathize with your hesitation, but making the code test-able is valuable here.  :)